src/table/src/table/scan.rs

```diff
@@ -64,14 +69,60 @@ impl RegionScanExec {
         if num_output_partition == 0 {
             num_output_partition = 1;
         }
+
+        let metadata = scanner.metadata();
+        let mut pk_columns: Vec<PhysicalSortExpr> = metadata
+            .primary_key_columns()
+            .map(|col| {
+                Ok(PhysicalSortExpr::new(
+                    Arc::new(Column::new_with_schema(
+                        &col.column_schema.name,
+                        &arrow_schema,
+                    )?) as _,
+                    SortOptions {
+                        descending: false,
+                        nulls_first: false,
+                    },
+                ))
+            })
+            .collect::<DfResult<Vec<_>>>()?;
+        let ts_col = PhysicalSortExpr::new(
+            Arc::new(Column::new_with_schema(
+                &metadata.time_index_column().column_schema.name,
+                &arrow_schema,
+            )?) as _,
+            SortOptions {
+                descending: false,
+                nulls_first: false,
+            },
+        );
+
+        let eq_props = match request.distribution {
+            Some(TimeSeriesDistribution::PerSeries) => {
+                pk_columns.push(ts_col);
+                EquivalenceProperties::new_with_orderings(
+                    arrow_schema.clone(),
+                    &[LexOrdering::new(pk_columns)],
+                )
+            }
+            Some(TimeSeriesDistribution::TimeWindowed) => {
+                pk_columns.insert(0, ts_col);
+                EquivalenceProperties::new_with_orderings(
+                    arrow_schema.clone(),
+                    &[LexOrdering::new(pk_columns)],
+                )
+            }
```

Comment:
Is this ordering different from the time window distribution? Since in the same time window, we sort data in `time series, timestamp` order now. But we didn't implement this distribution so this may not be a problem.

Reply:
I left a todo. I expect to handle this via time window distribution in the future